index.js

```diff
@@ -2613,6 +2613,42 @@
   }
   S.ifElse = def('ifElse', {}, [Pred(a), Fn(a, b), Fn(a, b), a, b], ifElse);
 
+  //# when :: (a -> Boolean) -> (a -> b) -> a -> b
```

Comment:
> I'm not sure on which version would be the most useful, but we should choose one which is "correct"

Absolutely! The pull request as it currently stands is not correct. I'm currently leaning towards the Ramda version, but I have a suspicion that I'm missing some deeper truth. This is a vague statement, I realize. ;)